.github/docs/assets/css/academicons-1.7.0/README.md

```
@@ -0,0 +1,15 @@
+Academicons
+===========
+Academicons is an icon font for academics. It aims to supplement other packages (such as Font Awesome) with icons for academic websites. Go to http://jpswalsh.github.io/academicons to view the full icon set along with instructions for their use.
+
+##License
```

markdownlint-cli2 (0.20.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)